pennylane/qnn/keras.py

```diff
@@ -370,3 +370,15 @@ def input_arg(self):
         `Layer <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer>`__. Set to
         ``"inputs"``."""
         return self._input_arg
+
+    def get_config(self):
+        """Returns `__init__` args used to construct this instance, allowing
+        serialization of models contianing `KerasLayer`.
```

Comment:
```suggestion
        serialization of models containing `KerasLayer`.
```

Reply:
Thanks for spotting this. I'll fix it in the next commit.